src/cheri_insts.sail

```diff
@@ -754,10 +755,8 @@ function clause execute (CUnseal(cd, cs1, cs2)) = {
   let permitted = cs2_val.tag
                 & isCapSealed(cs1_val)
                 & not(isCapSealed(cs2_val))
-                & (cs2_addr == unsigned(cs1_val.otype))
-                & cs2_val.permit_unseal
-                & (cs2_addr >= cs2_base)
-                & (cs2_addr < cs2_top);
+                & inCapBounds(cs2_val, zero_extend(cs1_val.otype), 1)
```

Comment:
I like this but you should also delete `getCapBounds` above.

Perhaps not for this PR but we could make the same change in `CSeal`.